pkg/apis/policy/types.go

```diff
@@ -66,7 +66,7 @@ type PodDisruptionBudgetSpec struct {
 	// Clients making eviction decisions should disallow eviction of unhealthy pods
 	// if they encounter an unrecognized policy in this field.
 	//
-	// This field is alpha-level. The eviction API uses this field when
+	// This field is beta-level. The eviction API uses this field when
 	// the feature gate PDBUnhealthyPodEvictionPolicy is enabled (disabled by default).
```

Comment:
```suggestion
    // the feature gate PDBUnhealthyPodEvictionPolicy is enabled (enabled by default).
```

@ravisantoshgudimetla these need to be updated as well

Reply:
Thank you for catching it @atiratree. Opened https://github.com/kubernetes/kubernetes/pull/115914 to address the same.